.gitignore

```diff
@@ -9,6 +9,7 @@
 .deps/
 .mypy_cache/
 __pycache__/
+cscope*
```

Comment:
what is this?

Reply:
Just like there is an exclusion for `vim` standard `*.swp` files above, this is an exclusion for `vim` standard `cscope` files.

Reply:
it's not related to this PR, so at the very least add a separate commit for it